src/runner/system_menu_helper.cpp

```diff
@@ -0,0 +1,127 @@
+#include "pch.h"
+#include "system_menu_helper.h"
+
+#include <interface/powertoy_module_interface.h>
+
+namespace {
+  constexpr int KSeparatorPos = 1;
+  constexpr int KNewItemPos = 2;
+
+  int GenerateItemId() {
+    static int generator = 0xDEADBEEF;
```

Comment:
Some more unique magic number would be better, DEADBEEF is kinda well-known

Reply:
Oh, and make it `unsigned`, so if it overflows it won't trigger undefined behavior.

Reply:
I can put 0x70777479 (`PWTY` string converted to HEX). No better idea.